lib/active_interactor/organizer/perform.rb

```diff
@@ -76,6 +76,7 @@ def merge_contexts(contexts)
       def perform_in_order
         self.class.organized.each do |interface|
           result = execute_interactor_with_callbacks(interface, true)
+          context_fail! if result&.failure?
           context.merge!(result) if result
         end
       rescue ActiveInteractor::Error::ContextFailure => e
```

Comment:
Rubocop does not catch the maximum `Cognitive Complexity` defined in CodeClimate. I managed to debug it by adding the following to `.rubocop.yml`

```yaml
Metrics/CyclomaticComplexity:
  Enabled: true
  Max: 6
```

Error:
```ruby
Method perform_in_order has a Cognitive Complexity of 7 (exceeds 5 allowed)
```